packages/foreman/rubygem-journald-native/rubygem-journald-native.spec

```diff
@@ -62,32 +62,34 @@ cp -pa .%{gem_dir}/* \
         %{buildroot}%{gem_dir}/
 
 mkdir -p %{buildroot}%{gem_extdir_mri}
-cp -a .%{gem_extdir_mri}/{gem.build_complete,*.so} %{buildroot}%{gem_extdir_mri}/
+%if 0%{?scl:1}
+  cp -a .%{gem_extdir_mri}/{gem.build_complete,*.so} %{buildroot}%{gem_extdir_mri}/
+%else
+  cp -a ./usr/share/gems/gems/%{gem_name}-%{version}/ext/journald_native/journald_native.so %{buildroot}%{gem_extdir_mri}/
```

Comment:
Using `%{gem_instdir}` should be equal to `/usr/share/gems/gems/%{gem_name}-%{version}`

Reply:
Sure.